packages/cursorless-everywhere-talon/src/registerCommands.ts

```diff
@@ -0,0 +1,59 @@
+import { CURSORLESS_COMMAND_ID } from "@cursorless/common";
+import type { CommandApi } from "@cursorless/cursorless-engine";
+import { Context, actions } from "talon";
+import type { TalonJsIDE } from "./ide/TalonJsIDE";
+
+const ctx = new Context();
+
+let ide: TalonJsIDE | undefined;
+let commandApi: CommandApi | undefined;
+
+ctx.matches = `
+not app: vscode
```

Comment:
@pokey What do we do with this matcher? For this to be valid with other editors we probably need a Cursorless everywhere tag instead?